firmware/ventilator-controller-stm32/Core/Inc/Pufferfish/Driver/BreathingCircuit/ControlLoop.h

```diff
@@ -16,6 +16,7 @@
 #include "Pufferfish/Driver/I2C/SFM3019/Sensor.h"
 #include "Pufferfish/HAL/Interfaces/PWM.h"
 #include "Pufferfish/Util/Timeouts.h"
+#include "SensorAlarmsService.h"
```

Comment:
`ControlLoop.h` should not depend on (or be aware of) the existence of a `SensorAlarmsService.h`. Here are two ways we could remove this dependency:

1. Move the declaration of `SensorConnections` into this file. After all, it's part of the public interface for the `ControlLoop` classes.
2. Move the declaration of `SensorConnections` into a new file, e.g. `Sensors.h`, which you include from both `ControlLoop.h` and `SensorAlarmsService.h`.

Reply:
Added Sensors.h and included in `ControlLoop` and `SensorAlarmsService`